k8s-multus/scripts/node01.sh

```diff
@@ -17,10 +17,11 @@ else
 kubectl --kubeconfig=/etc/kubernetes/admin.conf create -f /tmp/flannel.yaml
 fi
 
-kubectl --kubeconfig=/etc/kubernetes/admin.conf create -f /tmp/kubernetes-multus.yaml
-kubectl --kubeconfig=/etc/kubernetes/admin.conf create -f /tmp/multus.yaml
+kubectl --kubeconfig=/etc/kubernetes/admin.conf create -f /tmp/cna/namespace.yaml
+kubectl --kubeconfig=/etc/kubernetes/admin.conf create -f /tmp/cna/network-addons-config.crd.yaml
+kubectl --kubeconfig=/etc/kubernetes/admin.conf create -f /tmp/cna/operator.yaml
+kubectl --kubeconfig=/etc/kubernetes/admin.conf create -f /tmp/cna/network-addons-config-example.cr.yaml
```

Comment:
what about

```shell
kubectl wait networkaddonsconfig cluster --for condition=Ready
```

Reply:
Yeah sure we can do it good point!

Reply:
@phoracek yep, that the images are all pre-fetched is the main requirement.

Reply:
Done :)